README.md

```diff
@@ -27,9 +27,9 @@ It was heavily inspired by the ftrace framework of the Linux kernel and
 the name uftrace stems from the combination of user and ftrace.
 
 It can record data from:
-- User space functions by dynamically patching functions at runtime, or with code
-  compiled with with `-pg`, `-fpatchable-function-entry`, `-mfentry`, `-mnop-mcount`,
-  `-finstrument-functions`, or using clang with `-fxray-instrument`.
+- Functions in the executable by dynamically patching functions at runtime using `-P.`.
+- Code compiled with `-pg`, `-f patchable-function-entry`, `-m fentry`, `-m nop-mcount`,
+  `-f instrument-functions` and by the clang-specific `-f xray-instrument`.
```

Comment:
Maybe we can remove the details about the compiler options and just say that function can come from either instrumented binary or regular binary.

Reply:
Thank you very much, done and pushed!